### PR TITLE
fix: Grade value is displayed in the modal

### PR DESCRIPTION
**Pull request by DmytroAlipov**
**TL;DR -**

`Original Grade` value is not displayed:
    gr_0

This value was available on the Palm release:
    gr_1

That's how it was in [Palm](https://github.com/DmytroAlipov/frontend-app-gradebook/blob/3644172d9471ed3299a9b5accece072b99a7582a/src/components/GradesView/EditModal/ModalHeaders.jsx#L65)

This is the data contained in `selectors.grades.useGradeData`:
```
{
    "courseId": "course-v1:RG+New-course-2+2024",
    "filteredUsersCount": 10,
    "totalUsersCount": 11,
    "gradeFormat": "percent",
    "showSpinner": false,
    "gradeOverrideCurrentEarnedGradedOverride": null,
    "gradeOverrideHistoryResults": [],
    "gradeOriginalEarnedGraded": 1,
    "gradeOriginalPossibleGraded": 1,
    "nextPage": null,
    "prevPage": null,
    "showSuccess": false,
}
```

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review

**Testing Instructions**
1. As instructor open `Instructor` tab
2. go to `Student admin`
3. click `View Gradebook`
4. Click on available grade:
    gp_3
5. Check `Original Grade` value

**After these fixes everything works correctly:**
gr_2

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @DmytroAlipov! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @arbrandes and @farhaanbukhsh! Would one of you be able to take a look at this? Thanks!

**Comment by farhaanbukhsh** [comment]
@mphilbrick211  I can have a look at this in this sprint :)

File: src/components/GradesView/EditModal/ModalHeaders.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const HistoryKeys = StrictDict({
  */
 export const ModalHeaders = () => {
   const { assignmentName, updateUserName } = selectors.app.useModalData();
-  const { currentGrade, originalGrade } = selectors.grades.useGradeData();
+  const { gradeOverrideCurrentEarnedGradedOverride, gradeOriginalEarnedGraded } = selectors.grades.useGradeData();
```
Comment (arbrandes) [inline src/components/GradesView/EditModal/ModalHeaders.jsx:25]: These are horribly long variable names, but yeah, not much we can do.

**Review by arbrandes — APPROVED** [review]
Looks good, thanks!

**Comment by openedx-webhooks** [comment]
@DmytroAlipov 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/390&entry.1484655318=https://github.com/DmytroAlipov&entry.752974735=2024-04-21+11:03&entry.1917517419=2024-05-01+15:37&entry.2133058324=Yes) so we can improve your experience in the future.
